justfile

```diff
@@ -15,10 +15,14 @@ check:
 
 # Run all the tests.
 test language="[rust|python]" : (_run_lang language \
-        "HUGR_TEST_SCHEMA=\"1\" cargo test --all-features" \
+        "HUGR_TEST_SCHEMA=\"1\" cargo test --features extension_inference" \
         "poetry run pytest"
     )
 
+# Run all proptests
+proptest:
+    cargo test --feature proptest '::proptest'
```

Comment:
`--feature proptest` is on by default, no? So the `'::proptest'` here causes the tests to execute when `cargo test` (/ `cargo test --feature proptest`, which should be equivalent) does not?